### PR TITLE
add delete vector mask command

### PR DESCRIPTION
**Pull request by pineapplespatula**
adapter change for the fix for issue https://github.com/adobe-photoshop/spaces-design/issues/2669

### CODE REVIEW

**Comment by pineapplespatula** [comment]
with @iwehrman  on PTO I'm sending this one your way @mcilroyc

File: src/lib/vectorMask.js [hunk 1]
```diff
@@ -98,16 +98,38 @@ define(function (require, exports) {
 
 
     /**
+     * Delete the current vector mask on the current layer
+     * 
+     * @return {PlayObject}
+     */
+    var deleteVectorMask = function () {
+        var vectorMaskRef = {
```
Comment (mcilroyc) [inline src/lib/vectorMask.js]: Since this is all static, it could be initialized once at module load time instead of each function call.

**Comment by mcilroyc** [comment]
I have one gripe, but otherwise I'd merge it.

**Comment by pineapplespatula** [comment]
@mcilroyc  i refactored a little more

**Comment by mcilroyc** [comment]
ok

**Pull request merged**